src/rpc/net.cpp

```diff
@@ -1016,6 +1016,55 @@ static RPCHelpMan sendmsgtopeer()
     };
 }
 
+static RPCHelpMan getaddrmaninfo()
+{
+    return RPCHelpMan{"getaddrmaninfo",
+                      "\nProvides information about the node's address manager by returning the number of "
+                      "addresses in the `new` and `tried` tables and their sum for all networks.\n"
+                      "This RPC is for testing only.\n",
+                      {},
+                      RPCResult{
+                              RPCResult::Type::OBJ_DYN, "", "json object with network type as keys",
+                              {
+                                      {RPCResult::Type::OBJ, "network", "the network (" + Join(GetNetworkNames(), ", ") + ")",
```

Comment:
nit: This is missing the `all_networks` key in the RPC help.

Can add a fix for this in https://github.com/bitcoin/bitcoin/pull/28523 as this touches the line.

Reply:
included in #28523

Reply:
done in #28565